content/docs/goFSH/API/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "API"
+weight: 25
+---
+
+GoFSH exposes a `fhirToFsh` function that can be used to convert FHIR to FSH.
+
+## Syntax
+```javascript
+fhirToFsh(fhir[, options])
+```
+
+## Parameters
+`fhir` - An array of FHIR resources, represented either as strings or JSON.
+
+`options` - An object which can have any combindation of the following attributes:
+* `dependencies` - An array of strings used to specify dependencies required for processing the FHIR. Dependencies should use the format `<packageId>#<version>` (example: `hl7.fhir.us.core#3.0.1`).
```

Comment:
Yikes, so after reading the SUSHI API documentation just now, it's really clear that the two APIs require different formats for the incoming `dependencies` option.  I think we need to align these.  Since we're aiming for Monday releases, I guess we don't need to align them _right now_, but we should make a goal to align them soon.

And since they'll already have been released, we may need to support both formats in at least one of them (to maintain backward compatibility).

Reply:
I'm not so sure. If we align how the APIs take dependencies, then we are making it so one or the other of the APIs isn't really aligned with how the corresponding CLI takes dependencies.

Reply:
But I am going to update this documentation so that the preferred format is `<packageId>@<version>`, since that is how GoFSH takes dependencies.

Reply:
OK.  As I said, we can leave it as-is for now.  And I guess I'll think more about those implications then.